README.md

```diff
@@ -88,6 +174,7 @@ Additional development notes can be found [here](DEV_NOTES.md)
 * Tim Kinnel: https://github.com/timmykat
 * Dave Mayo: https://github.com/pobocks
 * Anita Patel: http://github.com/apatel
+* Kathryn (Katie) Amaral: https://github.com/ktamaral
```

Comment:
We've been arranging these in alpha order by last name to avoid assumption of preference - so your name should be first rather than last in list (or we should come up with a different metric but I think that can wait till literally forever)